brick/lib.py

```diff
@@ -107,10 +108,47 @@ def get_relative_config_path(target):
     return os.path.relpath(get_config_path(target), start=ROOT_PATH)
 
 
+def expand_brick_environment_variables(before_expansion: str) -> str:
```

Comment:
Seeing as this is now becoming quite specific and it's hard to read/understand regex, would you mind adding a comment here (or somewhere else in the code) about about usage and include an example of how this can be used? :)

**EDIT:** Just saw the test now which does exactly this, but maybe we could add a comment pointing to the test for examples of usage?

Reply:
https://github.com/tmrowco/brick/pull/64/commits/6bc22a7a403af31e039817cf900646bcf91391c7